pkg/imageregistry/imageregistry.go

```diff
@@ -134,8 +134,23 @@ func (builder *Builder) GetManagementState() (v1.ManagementState, error) {
 	return builder.Object.Spec.ManagementState, nil
 }
 
+// GetStorageConfig fetches imageRegistry Storage configuration.
+func (builder *Builder) GetStorageConfig() (imageregistryv1.ImageRegistryConfigStorage, error) {
```

Comment:
Please replace return by *imageregistryv1.ImageRegistryConfigStorage

Reply:
it doesn't work well after, I need value and not list of the pointers

Reply:
Can you please explain what the point of getting empty imageregistryv1.ImageRegistryConfigStorage{} instead of nil? 

Below you can see an example of refactored function:

```
// GetStorageConfig fetches imageRegistry Storage configuration.
func (builder *Builder) GetStorageConfig() (*imageregistryv1.ImageRegistryConfigStorage, error) {
	if valid, err := builder.validate(); !valid {
		return nil, err
	}

	glog.V(100).Infof("Getting imageRegistry Storage configuration")

	if !builder.Exists() {
		return nil, fmt.Errorf("imageRegistry object doesn't exist")
	}

	return &builder.Object.Spec.Storage, nil
}

```

Reply:
I prefer to leave it like this

Reply:
I cannot use *imageregistryv1.ImageRegistryConfigStorage. It making a problem with the results comparing after